### ISSUE TITLE
JAWS does not announces any instant message upon adding fluent component

### ISSUE THREAD

**Issue by junchen1** [microsoft/fluentui#17873]
Environment:
Windows OS version: 2004 (Build 21313.1000)
Edge dev version:  90.0.818.0(Official build) dev (64-bit)
Url: https://outlook-sdf.office.com/mail/inbox
AT Used: JAWS
JAWS Version:202.2011.16 ILM
Pre-requisites:
Open the URL: https://outlook-sdf.office.com/ and login with valid credentials.
Repro Steps:
Navigate to 'New Message' button and navigate to 'attach' button and activate it.
 Select any file or image from 'Browse my Computer' and activate 'Open' button and listen to JAWS announcement.
Actual Result:
JAWS does not announces any instant message upon adding any attachment.

Narrator Behaviour:
Upon adding any attachement Narrator announces'attahcment added successffully' success message instantly.

Expected Result:
JAWS should announce 'Attachement added successfully' success message instantly upon adding any attachment.

**Comment by ecraig12345** [comment]
Thanks for reporting this, but we'll need an isolated repro (not a repro in a product) to investigate. You can start from https://aka.ms/fluentpen or click "Export to CodePen" on one of the [component examples](https://developer.microsoft.com/en-us/fluentui#/controls/web), then edit it to be more like your scenario.

**Comment by junchen1** [comment]
import { Announced as FabricAnnounced, IAnnouncedProps } from '@fluentui/react/lib/Announced';
<FabricAnnounced {...props} />;

Announced component doesn't work with JAWS

**Comment by ecraig12345** [comment]
@junchen1 Next time can you please make a *complete, live* repro using https://aka.ms/fluentpen, https://aka.ms/fluentsandbox, or starting from one of the website examples? There's still not enough info to understand exactly how you're using the components.

For this time, I made a codepen including what I can understand so far. Can you modify this to reflect the rest of your scenario? https://codepen.io/ecraig12345/pen/OJpMwQd?editors=0010

**Comment by junchen1** [comment]
I don't know how to change your code.
Here is the what we use.
const { 
  Announced,
  DefaultButton
} = window.FluentUIReact;

const Content = () => {
  // NOTE: This template uses @fluentui/react version 8.
  // To use version 7, start from https://aka.ms/fabric7pen
  const [announcedComponentMessage, setAnnouncedComponentMessage] = React.useState(false);
  return (
    <div>
      <DefaultButton onClick={()=>setAnnouncedComponentMessage('test')}>cick to announce</DefaultButton>
      {announcedComponentMessage && (
        <Announced message={`${announcedComponentMessage}`} >
          
        </Announced>
      )}
    </div>
  );
};

ReactDOM.render(
  <Content />, 
  document.getElementById("content")
);

**Comment by flyingbee2012** [comment]
@ecraig12345 I modified your code as below. As you can see, the logic is that it should announce "test test test" when we click on it first time, and nothing 2nd time, and then "test test test" 3rd time and nothing, so on and so forth. Everything works as expected with Narrator, but the behavior is not consistent when I tried with Jaws. Can you experiment with Jaws it as well?

```
const { 
  Announced,
  DefaultButton
} = window.FluentUIReact;

const Content = () => {
  // NOTE: This template uses @fluentui/react version 8.
  // To use version 7, start from https://aka.ms/fabric7pen
  const [isShown, setIsShown] = React.useState(false);
  return (
    <div>
      <DefaultButton onClick={()=>setIsShown(!isShown)}>show content</DefaultButton>
      {isShown && (
        <Announced message='test test test'>
          {/*add the content to announce here*/}
        </Announced>
      )}
    </div>
  );
};

ReactDOM.render(
  <Content />, 
  document.getElementById("content")
);
```
